lib/active_admin/async_exporter/config.rb

```diff
@@ -18,7 +18,7 @@ class Config
       def initialize
         @aws_bucket_name = nil
         @aws_bucket_folder_path = nil
-        @disk_folder_path = nil
+        @disk_folder_path = './csv'
```

Comment:
maybe inside tmp like `./tmp/csv` ?

Reply:
As santib said we should use the tmp folder and maybe rename the folder to be the gem's name.

```suggestion
        @disk_folder_path = Rails.root.join('tmp/activeadmin-async_exporter')
```